packages/core/package.json

```
@@ -702,7 +702,7 @@
                     "type": "webview",
                     "id": "aws.AmazonQChatView",
                     "name": "%AWS.amazonq.chat%",
-                    "when": "!isCloud9 && !aws.isSageMaker"
+                    "when": "!isCloud9 && !aws.isSageMaker && !aws.dev.newLoginUxEnabled && aws.explorer.showAuthView"
```

"enabled" is never needed in a setting name. 
```suggestion
                    "when": "!isCloud9 && !aws.isSageMaker && !aws.dev.newLoginUx && aws.explorer.showAuthView"
```